core/src/main/resources/lib/layout/copyButton/copyButton.js

```diff
@@ -27,12 +27,12 @@ Behaviour.specify(
               copyButton,
             );
           });
-      } else {
-        hoverNotification(
-          "Copy is only supported with a secure (HTTPS) connection",
-          copyButton,
-        );
-      }
-    });
+      });
+    } else {
+      copyButton.disabled = true;
+      copyButton.removeAttribute("tooltip");
+      const parent = copyButton.parentElement;
+      parent.setAttribute("tooltip", parent.dataset.messageInsecure);
```

Comment:
Out of curiosity, why add the `span` when you could put the `data-` attribute on the button itself?

Reply:
The tooltip attribute of a button is not working with tippy when the button is disabled. It will only show the browser built-in tooltip. So I need the span around the button to be able to show a tippy tooltip anyway